### DISCUSSION TITLE
How to represent tips for cmake after installation?

### DISCUSSION THREAD

**Discussion by playgithub** [microsoft/vcpkg#25550] [General]
e.g.
```cmd
vcpkg install WTL
...
wtl is header-only and can be used from CMake via:
    find_path(WTL_INCLUDE_DIRS "atlapp.h")
    target_include_directories(main PRIVATE ${WTL_INCLUDE_DIRS})
```

The tips after installation are useful, anyway to represent it?

**Comment by Osyotr** [comment]
vcpkg will print whatever is located in `usage` file:
https://github.com/microsoft/vcpkg/blob/4826ede84084395b0f87476e48ff6d4070eba0e4/ports/zlib/portfile.cmake#L46
https://github.com/microsoft/vcpkg/blob/4826ede84084395b0f87476e48ff6d4070eba0e4/ports/zlib/usage#L1-L4

**Reply by playgithub** [reply]
> vcpkg will print whatever is located in `usage` file

For port `wtl`, no file `usage`, but tips still output.

**Reply by Osyotr** [reply]
Sorry, I misunderstood your question. I though you wanted to add usage information to a port. Ignore my answer.

**Comment by dg0yt** [comment]
The usage will also be printed again for installed ports when you call `vcpkg install <port>`.

But be warned that there is also heuristical output which can be wrong (#20190).
